addons/event/views/event_event_views.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0"?>
 <odoo><data>
+    <record id="event_barcode_action_main_view" model="ir.actions.client">
```

Comment:
:socks: could be put into a 'ir_actions_client_data_xml' data file in data

Reply:
@tde-banana-odoo there doesn't exist any file with such name, and i don't think creating it would be better option than leaving this here